libraries/cli/src/config.cpp

```diff
@@ -168,6 +169,12 @@ Config::Config(int argc, const char* argv[]) {
       fs::create_directories(data_dir);
     }
 
+    // if there is not chain id it is a test
```

Comment:
Why do we create config updater only if this condition pass ? Shouldnt it be created every time ?

Reply:
read the comment above you line. As test do not have chain id, I can not apply update